workloads/files/workload-http-script-cm.yml

```diff
@@ -301,8 +301,15 @@ data:
       local oc_whoami=`oc whoami`
 
       test "$HTTP_TEST_LOAD_GENERATOR_NODES" || {
-        echo "Not (un)labelling nodes, load generator nodes unspecified."
-        return 0
+        echo "Load generator nodes unspecified."
+        if [ $HTTP_TEST_LOAD_GENERATORS -eq 1 ] ; then
+          HTTP_TEST_LOAD_GENERATOR_NODES=$(oc get nodes --no-headers | grep "workload" |  awk '{print $1}')
```

Comment:
Maybe add a check to see if workload node exists?

Reply:
if workload node doesn't exist it will run like it used to, use a worker node.

Reply:
Ack, thanks.